e2e/app.e2e.ts

```diff
@@ -151,5 +152,13 @@ describe('bterm launch', function() {
       .then(() => this.app.client.browserWindow.isMaximized())
       .then(result => expect(result).to.be.true);
   });
+
+  it('should open and close multi window of the app', () => {
+    let tmpApp = clone(this.app);
+    return this.app.client.waitUntilWindowLoaded()
+      .then(() => tmpApp.start())
+      .then(() => tmpApp.client.click('.close'))
+      .then(() => wait(1000))
```

Comment:
I think you need to tests here if the app is closed / is null? I cannot see any test here.